Polyclinic/Polyclinic.Grpc.Client/AppointmentGenerator.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Bogus;
+using Polyclinic.Grpc.Protos;
+
+namespace Polyclinic.Grpc.Client;
+
+/// <summary>
+/// Appointment contract generator using Bogus library.
+/// </summary>
+public class AppointmentGenerator
+{
+    private readonly Faker _faker = new();
+
+    /// <summary>
+    /// Generates one random appointment contract.
+    /// </summary>
+    /// <param name="maxPatientId">Maximum patient ID in database.</param>
+    /// <param name="maxDoctorId">Maximum doctor ID in database.</param>
+    public AppointmentResponse GenerateRandomContract(
+        int maxPatientId = 10,
+        int maxDoctorId = 10)
+    {
+        var hour = _faker.Random.Int(8, 18);
+        var minute = _faker.PickRandom(new[] { 0, 15, 30, 45 });
+
+        return new AppointmentResponse
+        {
+            PatientId = _faker.Random.Int(1, maxPatientId),
+            DoctorId = _faker.Random.Int(1, maxDoctorId),
+            AppointmentDate = _faker.Date.FutureDateOnly(1).ToString("yyyy-MM-dd") +
+                             $"T{hour:00}:{minute:00}:00",
+            RoomNumber = _faker.Random.Int(100, 500),
+            IsFollowUp = _faker.Random.Bool(0.3f) 
+        };
```

Comment:
1. `new[] {0, 15, 30, 45 }` лучше вынести в static readonly поле;
2. С датой проще сначала сгенерировать DateTime а затем привести к формату даты.
```suggestion
        var date = _faker.Date.Future(1);
        return new AppointmentResponse
        {
            PatientId = _faker.Random.Int(1, maxPatientId),
            DoctorId = _faker.Random.Int(1, maxDoctorId),
            AppointmentDate = new DateTime(
                date.Year,
                date.Month,
                date.Day,
                _faker.Random.Int(8, 18),
                _faker.PickRandom(_minutes),
                0).ToString("o"),
            RoomNumber = _faker.Random.Int(100, 500),
            IsFollowUp = _faker.Random.Bool(0.3f)
        };
```
Форматы даты и времени можете свой оставить, я просто для примера использовал "o". Если вам он не подходит используйте свой, только наверно лучше тоже формат в константу вынести.
<img width="473" height="62" alt="image" src="https://github.com/user-attachments/assets/8cc7b90c-c390-4d63-8630-0eb39dac9524" />